libgstc/python/pygstc/gstc.py

```diff
@@ -829,6 +829,26 @@ def pipeline_create(self, pipe_name, pipe_desc):
         parameters = self._check_parameters([pipe_name, pipe_desc], [str, str])
         self._send_cmd_line(['pipeline_create'] + parameters)
 
+    def pipeline_create_ref(self, pipe_name, pipe_desc):
```

Comment:
Aren't you missing unit tests in `tests/libgstc/python` for these new functions?

Reply:
`pygstc` checks are currently disabled: 4a4c80db